scripts/release.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-undef */
+/* eslint-disable camelcase */
+const Asana = require('asana')
+
+const ASANA_ACCESS_TOKEN = env.ASANA_ACCESS_TOKEN
+const commit = GITHUB_SHA
+const version = github.event.release.tag_name
+const releaseUrl = github.event.release.html_url
+const releaseNotes = github.event.release.body
+const templateTaskGid = '1200547430029363'
+const autofillProjectGid = '1198964220583541'
+const releaseSectionGid = '1200559726959935'
+const projectExtractorRegex = /\[\[project_gid=(\d+)]]\s/
+
+let asana
+
+const setupAsana = () => {
+    asana = Asana.Client.create({
+        'defaultHeaders': {
+            'Asana-Enable': 'new_user_task_lists'
+        }
+    }).useAccessToken(ASANA_ACCESS_TOKEN)
+}
+
+const duplicateTemplateTask = () => {
+    const duplicateOption = {
+        include: ['notes', 'assignee', 'subtasks', 'projects'],
+        name: `Autofill Release ${version}`,
+        opt_fields: 'html_notes'
+    }
+
+    return asana.tasks.duplicateTask(templateTaskGid, duplicateOption)
+}
+
+const run = async () => {
+    setupAsana()
+
+    console.info('Asana on. Duplicating template task...')
+
+    const { new_task } = await duplicateTemplateTask()
```

Comment:
Very minor, but worth passing `templateTaskGid` as a param here? Makes it slightly more flexible, but I don't feel strongly.